packages/client-adapters/src/adapters/lexicons/adapter.js

```diff
@@ -369,28 +369,34 @@ class AlpheiosLexiconsAdapter extends BaseAdapter {
     // if we still don't have a match, we can do a last ditch check without
     // any diacritics at all in those languages that support it
     if (!found) {
-      let lastAlt = []
+      const lastAlt = []

```

Comment:
I can never remember, but don't we need to define arrays with let if new values are going to be pushed to them?

Reply:
(if so, this is my mistake for not including the necessary eslint here I know...)

Reply:
yes - here should be special es-lint commands
I didn't touch this package during development - that's why I decided not to fix it bfore review - I would add this es-lint rule

Reply:
thank you!